pkg/model/components/etcdmanager/options.go

```diff
@@ -52,8 +51,7 @@ func (b *EtcdManagerOptionsBuilder) BuildOptions(o interface{}) error {
 			etcdCluster.Backups = &kops.EtcdBackupSpec{}
 		}
 		if etcdCluster.Backups.BackupStore == "" {
-			base := clusterSpec.ConfigBase
-			etcdCluster.Backups.BackupStore = urls.Join(base, "backups", "etcd", etcdCluster.Name)
```

Comment:
Removing `backups/etcd/{etcd-cluster}` here as it is already set by Location in pkg/model/components/etcdmanager/model.go